civictechprojects/views.py

```diff
@@ -110,6 +114,8 @@ def group_edit(request, group_id):
 
 # TODO: Pass csrf token in ajax call so we can check for it
 @csrf_exempt
+@api_view(['GET', 'POST'])
+@throttle_classes([AnonRateThrottle, UserRateThrottle])
 def group_delete(request, group_id):
```

Comment:
This operation is failing, and I'm not sure why.  It's definitely being sent with POST.

Reply:
Can you elaborate what you mean by "failing"? :) I just ran it and it seems to work:
```
xh POST http://127.0.0.1:8000/api/groups/delete/2/
HTTP/1.1 204 No Content
```